TreeTests/findMaximumBinaryTree.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Xunit;
+
+namespace TreeTests
```

Comment:
You don't need a separate test project for trees. Add tests to `DataStructures.Tests`.